sympy/printing/mathml.py

```diff
@@ -554,6 +554,9 @@ def multiply(expr, mrow):
             numer, denom = fraction(expr)
             if denom is not S.One:
                 frac = self.dom.createElement('mfrac')
+                print(expr)
```

Comment:
Remove this.

Reply:
Ohh sorry. That was by mistake. It was for testing purpose.